pkg/cmd/docgen/diagrams.go

```diff
@@ -822,6 +822,8 @@ var specs = []stmtSpec{
 	{name: "show_trace", stmt: "show_trace_stmt"},
 	{name: "show_transaction", stmt: "show_stmt", match: []*regexp.Regexp{regexp.MustCompile("'SHOW' 'TRANSACTION'")}},
 	{name: "show_users", stmt: "show_stmt", match: []*regexp.Regexp{regexp.MustCompile("'SHOW' 'USERS'")}},
+	{name: "split_index_at", stmt: "alter_split_index_stmt"},
```

Comment:
For consistency with `ALTER INDEX ... RENAME` you might want to apply the following config:

```
inline: []string{"table_name_with_index"}
replace: map[string]string{"qualified_name": "table_name", "'@' name": "'@' index_name"},
```

Reply:
Good point! Done.